### PR TITLE
Gcw 1467 on dispatch locations ui

### PR DESCRIPTION
**Pull request by namrataukirde**

### CODE REVIEW

File: app/controllers/items/partial_dispatch.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+export default Ember.Controller.extend({
+  item: null,
+  queryParams: ["orderPackageId"],
+  orderPackageId: null,
+  partialDispatchBackLinkpath: Ember.computed.localStorage(),
+
+  orderPackage: Ember.computed('orderPackageId', function(){
+    var orderPackage = this.get('store').peekRecord('OrdersPackage', this.get('orderPackageId'));
+    return orderPackage;
```
Comment (Darshan-Chauhan) [inline app/controllers/items/partial_dispatch.js]: Can we directly return **this.get('store').peekRecord('OrdersPackage', this.get('orderPackageId'));**? Is there any need for declaring variable?

File: app/controllers/items/partial_move.js [hunk 2]
```diff
@@ -32,6 +35,19 @@ export default Ember.Controller.extend({
       }else{
         this.transitionToRoute('items.search_location', item.id, { queryParams: { isPartialMove: true }});
       }
+    },
+    editPackagesLocation(packagesLocation){
```
Comment (Darshan-Chauhan) [inline app/controllers/items/partial_move.js]: In our app for declaring function we use following convention

`editPackagesLocation: function(packagesLocation) {
},`

File: app/controllers/items/partial_move.js [hunk 3]
```diff
@@ -32,6 +35,19 @@ export default Ember.Controller.extend({
...
+      this.set('isEditing', true);
+      packagesLocation.set('isEditing', true);
+    },
+
+    notNow(item){
```
Comment (Darshan-Chauhan) [inline app/controllers/items/partial_move.js]: ^

File: app/controllers/items/partial_move.js [hunk 4]
```diff
@@ -32,6 +35,19 @@ export default Ember.Controller.extend({
...
+      var editedPackagesLocations = item.get('packagesLocations').filterBy('isEditing', true);
+      editedPackagesLocations.forEach(packagesLocation => {
```
Comment (Darshan-Chauhan) [inline app/controllers/items/partial_move.js]: Can we rename ** packagesLocation** to **packageLocation** as it is single record?

File: app/templates/items/detail.hbs [hunk 5]
```diff
@@ -26,12 +26,25 @@
...
+          </div>
         {{/if}}
 
+        {{!--
+          {{#if (is-equal model.isSet model.setItem.hasSingleDesignation)}}
+            {{dispatch-set item=model order=model.designation autoDisplayOverlay=autoDisplayOverlay}}
+          {{else if (is-and model.hasOneDesignatedPackage (is-not model.isSet))}}
+            {{dispatch-set item=model order=model.designation package=model.hasOneDesignatedPackage autoDisplayOverlay=autoDisplayOverlay}}
+          {{/if}}
+        --}}
```
Comment (Darshan-Chauhan) [inline app/templates/items/detail.hbs]: Can we remove this if not needed?

File: app/controllers/items/partial_move.js (hunk 2 above)
Comment (trojanh) [inline app/controllers/items/partial_move.js]: In ECMA6, it is recommended that we use functionName(){..} and not functionName: function() {...}
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Classes

Comment (namrataukirde) [inline app/controllers/items/partial_move.js]: I am agree with @trojanh 

http://es6-features.org/#MethodProperties

**Pull request merged**